rcsb/utils/targets/CARDTargetProvider.py

```diff
@@ -95,6 +118,47 @@ def __reload(self, dirPath, **kwargs):
         # ---
         return oD, version
 
+    def __reloadOntology(self, dirPath, **kwargs):
```

Comment:
Just food for thought question, would it make sense for this ontology stuff to be in its own provider or is it dependent on the other data in this provider?

Reply:
I considered that too, but it seemed overkill for just 2-3 methods... What do you think?

Reply:
On second thought, I decided to pull that out into a separate class. Please take a look the updated changes.

Reply:
Looks good.